commands/pull_request.go

```diff
@@ -185,6 +187,18 @@ func pullRequest(cmd *Command, args *Args) {
 		}
 	}
 
+	if flagPullRequestShow {
+		pullRequests, err := client.GetPullRequestUrlsForHead(headProject, head)
+		utils.Check(err)
+
+		for _, pr := range pullRequests {
+			ui.Println(pr.HtmlUrl)
```

Comment:
This prints all results found.  There can only ever be one open PR for a given branch, so if there is more than 1 result, the rest will be closed PRs. Some questions:

1. Do we want to visually indicate open/closed status?
2. How will we order the results?
3. What about past PRs for whom is the branch name is identical to the current branch name, but they were completely different branches at different times that happened to share the same name? Will we try to filter those out from results?
4. Do we want to consider only showing the open PR, or nothing if there are no open PRs?